src/dxtbx/model/detector_helpers.py

```diff
@@ -11,6 +11,8 @@
 except ImportError:
     sklearn = None
 
+from xfel.cftbx.detector.metrology import get_projection_matrix
```

Comment:
We should avoid a dependence on `xfel` if possible as `dxtbx` should remain an upstream dependency for DIALS and cctbx.xfel. That function looks fairly basic so perhaps could just be reproduced here, or the original moved to e.g. `scitbx`